metrictank-sample.ini

```diff
@@ -286,6 +288,8 @@ auth = false
 username = cassandra
 # password for authentication
 password = cassandra
+# sets whether to create or check for the cassandra keyspaces
```

Comment:
when this file is in final shape, run scripts/sync-configs.sh to propagate these new settings to all other config files in the repo (get familiar with the vimdiff keybinds if you're not already and this will become easy)

for docker/docker-cluster/metrictank.ini, that's the file for the dev cluster setup, with multiple instances using the same config, these options should be false, and then put in docker/docker-cluster/docker-compose.yml the environment variables for 1 of the instances (eg metrictank0) like so:

```
MT_CASSANDRA_CREATE_KEYSPACE: "true"
MT_CASSANDRA_IDX_CREATE_KEYSPACE: "true"
```